### PR TITLE
Flatpak related things

### PR DESCRIPTION
**Pull request by heschy2**
1.: I replaced "Flatpack" with "Flatpak" because writing it with ck is wrong.
2.: I added a Flatpak section to the german installation instructions.

Solves #185

### CODE REVIEW

**Comment by nathanlesage** [comment]
Thanks for this! There are only a few minor typos in the German documentation update, and I used informal speech ("du") everywhere else -- after these minor things are fixed we can immediately merge this! Thanks again for your work.

**Comment by heschy2** [comment]
> There are only a few minor typos in the German documentation update

Oh, sorry for the oversight. I tried to correct them, but I didn't found them, could you please explain which typos you are referring to?

File: docs/de/install.md [hunk 1]
```diff
@@ -41,6 +41,18 @@ Es gibt zwei Pakete für Arch Linux, die von der Community gepflegt werden. Das
 
 Zettlr ist als `AppImage` Bundle für Linux-Systeme verfügbar und kann auf unserer [Downloadseite](https://www.zettlr.com/download) heruntergeladen werden. Für mehr Informationen über das Installieren von AppImage Bundles empfehlen wir den offiziellen [Leitfaden auf appimage.org](https://appimage.org/).
 
+### Flatpak
+
+Zettlr ist als `Flatpak` für Linux-Systeme verfügbar und kann auf [Flathub](https://flathub.org/apps/com.zettlr.Zettlr) heruntergeladen werden. Außerdem ist es von Vorteil wenn du auch die `texlive`-Erweiterung für Flatpak installierst, da diese die Verwendung von LaTeX ermöglicht. LaTeX wird von Zettlr benötigt um z.b. Dateien als PDF zu exportieren.
```
Comment (nathanlesage) [inline docs/de/install.md:46]:
```suggestion
Zettlr ist als `Flatpak` für Linux-Systeme verfügbar und kann auf [Flathub](https://flathub.org/apps/com.zettlr.Zettlr) heruntergeladen werden. Außerdem ist es von Vorteil, wenn du auch die `texlive`-Erweiterung für Flatpak installierst, da diese die Verwendung von LaTeX ermöglicht. LaTeX wird von Zettlr benötigt um z.B. Dateien als PDF zu exportieren.
```

File: docs/de/install.md [hunk 2]
```diff
@@ -41,6 +41,18 @@ Es gibt zwei Pakete für Arch Linux, die von der Community gepflegt werden. Das
...
+
+Die Installation von Zettlr kann auch komplett auf der Konsole ausgeführt werden, wenn Flatpak bereits [eingerichtet wurde](https://flathub.org/setup):
+
+```bash
+flatpak install flathub com.zettlr.Zettlr org.freedesktop.Sdk.Extension.texlive
+```
+> Um LaTeX nicht zu installieren entferne einfach `org.freedesktop.Sdk.Extension.texlive` aus dem Befehl. Beachte aber bitte, dass du dann nicht in der Lage bist Funktionen zu verwenden die LaTeX benötigen.
```
Comment (nathanlesage) [inline docs/de/install.md:53]:
```suggestion
> Um LaTeX nicht zu installieren, entferne `org.freedesktop.Sdk.Extension.texlive` aus dem Befehl. Beachte aber bitte, dass du dann nicht in der Lage bist, Funktionen zu verwenden, die LaTeX benötigen.
```

**Review by nathanlesage — APPROVED** [review]
Now, sorry I'm a comma enthusiast 🙈

**Comment by heschy2** [comment]
Okay, I think I applied the changes you suggested, but I am not sure if I did it right (this is the first Pull-Request I ever made, I litteraly had to google how to create one in the first place 🙈)

**Comment by nathanlesage** [comment]
Haha no worries -- we all have to start somewhere, and from what I can see you've been handling the process exceptionally well! It all looks good to me now, so I'll merge it in! Thanks again for your work, and as always, if you find anything else, open another PR! :)

**Pull request merged**